app/components/ui/molecules/StaffGrid.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Flex, Box } from 'grid-styled'
+import { th } from '@pubsweet/ui-toolkit'
+
+const Card = styled(Box)`
+  padding-top: ${th('space.3')};
```

Comment:
I'm really sorry for leaving these comments in so many bits @jyenes but there's one final thing I've just noticed. This is creating spacing above the grid, as well as between the cards.

How about doing something similar to the x-axis padding?

I'm thinking something like:
```js
<Flex mb={-3} mx={-2}>
    {children.map((item, index) => (
      <Box mb={3} mx={2}
```

Reply:
don't worry @jsms90 good catch